output/.gitignore

```diff
@@ -0,0 +1 @@
+/model.joblib
```

Comment:
I guess dvc will update this automatically when the outputs are added to the file. Otherwise we have to edit this ourselves with the other files as well.

Reply:
wowoow it didnt?

Reply:
oops forgot about this, gonna add it. would `/*` work?

Reply:
yups